hack/make.ps1

```diff
@@ -0,0 +1,61 @@
+$DOCKER_PKG='github.com/docker/docker'
+$ErrorActionPreference = "Stop"
+
+# convert the .go-autogen bash script to ps1
+function ConvertGoAutogen {
+Write-Host "Converting hack\make\.go-autogen to hack\make\.go-autogen.ps1"
+(Get-Content hack\make\.go-autogen) `
+  -replace("rm -rf autogen", "if (Test-Path autogen) { rm -Recurse -Force autogen }
+	`$GITCOMMIT = ""$(git rev-parse --short HEAD)""
+	`$VERSION = ""$(cat VERSION)""
+	`$BUILDTIME = ""$(date)""
+	`$IAMSTATIC = ""false""
+	`$DOCKER_INITSHA1 = """"
+  `$DOCKER_INITPATH = """"
+  `$BASH_SOURCE = ""hack\make\.go-autogen.ps1""
+	") `
+  -replace("cat > dockerversion/version_autogen.go <<DVEOF", '$version_autogen = @"') `
+  -replace(":-true", '') `
+  -replace("DVEOF", '"@ | Out-File -Encoding "Ascii" dockerversion\version_autogen.go') `
+  -replace ("cat > autogen/winresources/resources.go <<WREOF", '$resources = @"') `
+  -replace("WREOF", '"@ | Out-File -Encoding "Ascii" autogen/winresources\resources.go') `
+  -replace("if \[", '# if [') `
+  -replace("\\$", "``") `
+  -replace("> /dev/null", '') `
+  -replace('^fi', '# fi') `
+  | Out-File -Encoding "Ascii" hack\make\.go-autogen.ps1
+
+	cp hack\make\.go-autogen.ps1 bundles
```

Comment:
I'll remove this line. This was to test whether all the environments were set. I got only `library-import` but then found out that I have to set the buildtags.